<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(54, 54, 54);color: rgb(177, 177, 177);
alternate-background-color: rgb(97, 97, 97);
selection-background-color: rgb(97, 97, 97);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(177, 177, 177); font-size:18pt;</string>
    </property>
    <property name="text">
     <string>You are logged in as:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_user">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(177, 177, 177); font-size:18pt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUsers">
    <property name="title">
     <string>Users</string>
    </property>
    <addaction name="actionChangePass"/>
    <addaction name="actionNew_User"/>
    <addaction name="actionAll_Users"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionPassReq"/>
   </widget>
   <addaction name="menuUsers"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChangePass">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Change password</string>
   </property>
  </action>
  <action name="actionNew_User">
   <property name="text">
    <string>New User</string>
   </property>
  </action>
  <action name="actionAll_Users">
   <property name="text">
    <string>All Users</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPassReq">
   <property name="text">
    <string>Pass Requirements</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
